### ISSUE TITLE
Is it possible to update this package to net.core

### ISSUE THREAD

**Issue by nafberger** [christophano/CsvHelper.Excel#7]
Hi Mr. Chris Ophano, Is it possible to update this package to net.core. please email me at nafberger@gmail.com. we are willing to pay

**Comment by christophano** [comment]
Hi Nafberger,
I do intend to update to support .net core, but I need the downstream dependencies to be updated first. CsvHelper has several betas for v3 which are on .net standard, but ClosedXml isn't there yet.

**Comment by smokja** [comment]
Any updates on further plans for .net standard / .net core? ClosedXml has updated to .Net Standard 2.0 if I didn't misread anything. [ClosedXml Post](https://github.com/ClosedXML/ClosedXML/issues/101)

**Comment by christophano** [comment]
Yes, this is still planned. I have been following the `ClosedXml` ticket. I've been busy, but I have a lot of the work done converting to `netstandard` already.
The problem is that `CsvHelper` only supports `netstandard` in versions after 3.0, which would require quite a bit of work to support, as 3.0 introduces several new types to implement, most of which is unrelated to Excel support.

**Comment by baoduy** [comment]
Nice work cheers. Awaiting you to update to .net core

**Comment by bryanllewis** [comment]
Any further progress on implementing .Net Standard / .Net Core support?  I've been using CsvHelper in a .NET Core 2.1 app and would love to be able to add Excel parsing as an option.

**Comment by MNF** [comment]
Is support .Net Core WIP? Do you have a branch for it? 
Can other people use it and be involved in migration?

**Comment by youngcm2** [comment]
@MNF 
You can try it in the active fork.

https://github.com/youngcm2/CsvHelper.Excel
